docs/troubleshooting.md

```diff
@@ -0,0 +1,17 @@
+| Generic Mounting Issue| Most of the common mount point issues are around permissions on both local mount point and the Cloud Storage bucket. It is highly recommended to retry with --foreground --debug_fuse --debug_fs --debug_gcs --debug_http flags which would provide much more detailed logs to understand the errors better and possibly provide a solution.                                                                                                                                                                                                                                                                      |
+| Mount successful but files not visible | Try mounting the gcsfuse with --implicit-dir flag. Read the semantics to know the reasoning.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| Mount failed with fusermount3 exit status 1 | It comes when the bucket is already mounted in a folder and we try to mount it again. You need to unmount first and then remount.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Mount failed with error: Current requires cgo or $USER set in environment | It comes when we try mounting by building the gcsfuse codebase. To fix this, build the gcsfuse package by enabling the CGO_ENABLED flag in the go env and then mount back.<ol type="a"><li> Check the current value using - go env command. </li><li>If it is unset, set this using - “export CGO_ENABLED=1” command. </li></ol>                                                                                                                                                                                                                                                                                  |
```

Comment:
you can also put go env and export CGO_ENABLED=1 in code blocks to make it consistent with other places.

Reply:
fixed

Reply:
fixed